src/java.base/share/classes/sun/security/provider/HSS.java

```diff
@@ -783,11 +779,18 @@ public String toString() {
         @Override
         protected void parseKeyBits() throws InvalidKeyException {
             byte[] keyArray = getKey().toByteArray();
-            if ((keyArray[0] != DerValue.tag_OctetString) || (keyArray[1] != keyArray.length -2)) {
-                throw new InvalidKeyException("Bad X509Key");
+            // Check less than minimum length to make sure this method works as expected
```

Comment:
s/Check less/Check if less/

or just delete the comment since it adds no useful information

Reply:
OK. My intent was to say this check is not final (because minimal length of an HSS/LMS public key is more than 12) but it is still necessary to prevent OOIBE in this method. I can see it is not very useful.